runtime/include/jvmti.h.m4

```diff
@@ -481,9 +481,9 @@ typedef struct {
 	unsigned int can_retransform_any_class : 1;
 	unsigned int can_generate_resource_exhaustion_heap_events : 1;
 	unsigned int can_generate_resource_exhaustion_threads_events : 1;
-	unsigned int can_generate_early_vmstart : 1;
-	unsigned int can_generate_early_class_hook_events : 1;
-	unsigned int : 5;
+ifelse(eval(JAVA_SPEC_VERSION >= 9), 1, ``	unsigned int can_generate_early_vmstart : 1;'', `	dnl')
+ifelse(eval(JAVA_SPEC_VERSION >= 9), 1, ``	unsigned int can_generate_early_class_hook_events : 1;'',`	dnl')
+ifelse(eval(JAVA_SPEC_VERSION >= 9), 1, ``	unsigned int : 5;'', ``unsigned int : 7;'')
```

Comment:
Why are the quotes doubled (``...'' instead of just `...')?
This would be easier to read if we use `[]` to quote text like other m4 files.
Can you express this group as a single `ifelse` rather than three?